new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -518,4 +561,85 @@ public static boolean deployApplication(String appName, String appLocation, Stri
                                           String username, String password, String target) {
     return true;
   }
+
+  // ------------------------ some utility method  -------------------------
+
+  /**
+   * Check running some command and returning expected value.
+   *
+   * @param command the command to run
+   * @param expectedValue the expected value should return
+   * @return true on success, false otherwise
+   */
+  public static boolean getExpectedResult(String command, String expectedValue) {
```

Comment:
change method name to indicate what its doing..may be runCommandAndCheckResultContainsString ? its too big ...see if you can have better name...

Reply:
changed the method name.